package.json

```diff
@@ -173,6 +169,7 @@
     "ajv": "^6.12.3",
     "array.prototype.flat": "^1.2.4",
     "array.prototype.flatmap": "^1.2.4",
+    "aws4-axios": "^2.4.9",
```

Comment:
is there a way to use a lighter weight library (maybe this?) https://github.com/mhart/aws4fetch. axios isn't huge but don't think we need it

Reply:
Thanks for the library, done 👍